File: java/src/Behavioral/Visitor/GameInfo.java
package Behavioral.Visitor;


//GameInfo.java - three of three concrete Visitees

public class GameInfo extends AbstractTitleInfo {  
   public GameInfo(String titleName) {
       this.setTitleName(titleName);
   }    
   
   public void accept(TitleBlurbVisitor titleBlurbVisitor) {
       titleBlurbVisitor.visit(this);
   }
}

File: csharp/src/Behavioral/Visitor/GameInfo.cs
namespace Behavioral.Visitor {


//GameInfo.java - three of three concrete Visitees

public class GameInfo : AbstractTitleInfo {  
   public GameInfo(string titleName) {
       this.setTitleName(titleName);
   }    
   
   override public void accept(TitleBlurbVisitor titleBlurbVisitor) {
       titleBlurbVisitor.visit(this);
   }
}

}
